demo/pydemolib.py

```diff
@@ -6,387 +6,30 @@
 Do not modify this file.
 """
 
```

Comment:
Perhaps it would be best to remove this `pydemolib.py` file altogether? Always keeping it in sync is a bit tedious, and as the C binary needs to be compiled anyway, running ctypesgen too shouldn't be much of a problem.